chore/release/release.go

```diff
@@ -0,0 +1,150 @@
+package chore
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/cli/go-gh/v2/pkg/api"
+	copyright "github.com/coreruleset/crs-toolchain/v2/chore/update_copyright"
+	"github.com/coreruleset/crs-toolchain/v2/context"
+	"github.com/go-git/go-git/v5"
+	"github.com/rs/zerolog/log"
+)
+
+var logger = log.With().Str("component", "release").Logger()
+
+func Release(context *context.Context, repositoryPath string, version *semver.Version, sourceRef string) {
+	remoteName := findRemoteName()
+	if remoteName == "" {
+		logger.Fatal().Msg("failed to find remote for coreruleset/coreruleset")
+	}
+	fetchSourceRef(remoteName, sourceRef)
+	branchName := fmt.Sprintf("v%d.%d.%d", version.Major(), version.Minor(), version.Patch())
+	createAndCheckOutBranch(context, branchName, sourceRef)
+	copyright.UpdateCopyright(context, version, uint16(time.Now().Year()), []string{"util/crs-rules-check/examples"})
```

Comment:
Maybe using a `const` defined on top instead of hardcoding it here? Hopefully we can remove the util directory soon...